.flake8

```diff
@@ -0,0 +1,6 @@
+[flake8]
+
+# E402 module level import not at top of file
+# gi.require_version() is required before later imports
+
+ignore = E402
```

Comment:
You seem to have an issue with adding no new line at EOF, you can fix this with `echo >> .flake8` the reason for this because when some tool like cat is used to view the file it prints the prompt on the same line as the last line in the file.

Reply:
Should every text file (including python scripts) have a new line at EOF?

Reply:
Yes.  Try to find out what caused you to miss that.  Look at files using tools that show the missing new line at EOF.  Prepare a file with and without the new line at EOF to test with.  If your tools are hiding that from you, report a bug against the tool, or use a better tool.  GitHub commit view does clearly warn us of this, which is one way we notice, but there are other ways as well.

On the other hand, don't go looking for the problem everywhere and make lots of pull requests to fix it.  The problem isn't that important.  Just be sure not to make the problem worse in a new pull request.

Reply:
That greatly helps, thanks. I will keep that in mind.